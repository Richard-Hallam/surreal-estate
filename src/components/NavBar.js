import React from 'react';
import '../styles/Navbar.css';

const NavBar = (props) => {
  (
    <div className="NavBar">
      <img src="src/logo.png" alt="surreal estate logo" />
    </div>
  );
};

export default NavBar;
