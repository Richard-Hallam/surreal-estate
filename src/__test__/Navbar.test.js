import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import NavBar from '../components/NavBar';


describe("NavBar", () => {
    const { asFragment } = render(<NavBar />);

    it("renders correctly", () => {
        expect(asFragment()).toMatchSnapshot();
    });

});