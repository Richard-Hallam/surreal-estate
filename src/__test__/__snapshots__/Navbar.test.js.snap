// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders correctly 1`] = `
<DocumentFragment>
  <div
    class="navbar"
  >
    <img
      alt="surreal estate logo"
      src="./logo"
    />
    <ul
      class="navbar-links"
    >
      <li
        class="navbar-links-item"
      >
        View Properties
      </li>
      <li
        class="navbar-links-item"
      >
        Add Properties
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
